xbmc/guilib/GUIFontTTF.cpp

```diff
@@ -60,9 +70,449 @@ constexpr int GLYPH_STRENGTH_BOLD = 24;
 constexpr int GLYPH_STRENGTH_LIGHT = -48;
 constexpr int TAB_SPACE_LENGTH = 4;
 
+constexpr std::chrono::hours JOB_RETRY_TIMEOUT{24};
```

Comment:
I'm not that familiar with C++ units, but can you initialize with `24h` to be type-correct?

Reply:
Checked, not works with it and will leave as 24.